docs/1___overview.adoc

```diff
@@ -156,3 +162,39 @@ The following changes were performed as part of the 1.0.1 maintenance release:
 * Minor typographical fixes (Issues #56, #57, #58)
 
 Note that there were no changes to the XML Schema files of the standard.
+
+=== Changes in 2.0.0
+
+SSP 2.0 is a major version release, all SSP 1.0 conforming artifacts are fully conforming to SSP 2.0.
+SSP 2.0 conforming artifacts are only compliant to the SSP 1.0 standard if they use a 1.0 version number and only use features available in SSP 1.0.
+
+The following changes were performed as part of the the 2.0.0 major release:
+
+* Support for FMI 3.0
+  ** Support for the new basic data types: Float32/64, and [U]Int8/16/32/64.
```

Comment:
"Binary" data type should be mentioned.

Reply:
That was already part of 1.0, so no new feature in SSP 2.0

Reply:
Have however added a remark to this effect so that people wont miss it.